notebooks/rendered/bigquery-basics.md

```diff
@@ -0,0 +1,232 @@
+
```

Comment:
No install steps? Possibly not needed, depending on where these notebooks are run.

Reply:
Added some installation steps in a readme